homeassistant/components/webhook/__init__.py

```diff
@@ -123,7 +153,15 @@ async def async_handle_webhook(
 
         if not network.is_local(remote):
             _LOGGER.warning("Received remote request for local webhook %s", webhook_id)
-            return Response(status=HTTPStatus.OK)
+            if webhook["local_only"]:
+                return Response(status=HTTPStatus.OK)
+            _LOGGER.warning(
```

Comment:
We have to ensure we log this only once for each webhook to prevent the log from being flooded.

Reply:
Done in [7d48999](https://github.com/home-assistant/core/pull/66494/commits/7d4899977d580e02e7919e80098896153ed0a48c)